03-bundling/06-vite/06-typescript/README.md

```diff
@@ -26,6 +26,20 @@ Install [Node.js and npm](https://nodejs.org/en/) (14.18+ / 16+) if they are not
   npm install typescript --save-dev
   ```
 
+- We'll switch to `"type": "module"` in `package.json`.:
+
+  _package.json_
+
+  ```diff
+  {
+    "name": "hello-vite",
+    "private": true,
+  + "type": "module",
+    "version": "0.0.0",
+  ```
+
+  This is for plugins to load dependencies as ESModules. [See this.](https://vite.dev/guide/troubleshooting#vite-cjs-node-api-deprecated)
```

Comment:
Unless we do this change it will throw warnings at the time of importing vite plugins saying CJS API is deprecated.